app/controllers/games_controller.rb

```diff
@@ -1,10 +1,13 @@
 class GamesController < ApplicationController
+  before_action :authenticate_user!
+
   def new
     @game = Game.new
   end
 
   def create
     @game = Game.create(game_params)
+    @game.players.create(game_id: @game.id, user_id: current_user.id, color: [:black, :white].sample)
```

Comment:
Yup LGTM. Once the second player joins, they will then be assigned to the other color and `populat_board!` will be called